spec/system/page_objects/components/sidebar.rb

```diff
@@ -23,6 +23,15 @@ def edit_custom_section(name)
         find(".sidebar-section-#{name.parameterize}").hover
         find(".sidebar-section-#{name.parameterize} button.sidebar-section-header-button").click
       end
+
+      def has_link?(name, href: nil)
+        return page.has_link?(name, href: "https://discourse.org") if href
+        page.has_link?(name)
+      end
```

Comment:
```suggestion
      def has_link?(name, href: nil)
        attributes = {}
        attributes[:href] = href if href
        page.has_link?(name, attributes)
      end
```

Perhaps something like this instead of having to repeat `page.has_link?` twice?